civic_europe/assets/scss/components/_custom_form.scss

```diff
@@ -79,6 +79,14 @@
         color: $brand-danger;
         font-size: $font-size-md;
     }
+
+    .help-block {
```

Comment:
How did you know it is this class? And where would I find 'bootstrap3/layout/help_text.html' (which is included in the checkboxselectmultiple.html)? This is all still magic to me :sweat_smile:

Reply:
Lot's of inspecting and grepping and trying it out. And I am not sure that really is the way to go, but works. :woman_shrugging: